test/http.js

```diff
@@ -159,64 +162,26 @@ describe('http', () => {
         }
       }
 
-      return http('http://example.com', req).then((response) => {
+      return http(req).then((response) => {
         expect(response.url).toEqual('http://example.com?anotherOne=one,two&evenMore=hi&bar=1%202%203')
         expect(response.status).toEqual(200)
-      }).then(fetchMock.restore)
+      })
     })
   })
 
   describe('serializeRes', function () {
     it('should serialize fetch-like response and call serializeHeaders', function () {
-      const headers = {
-        Authorization: ['Basic hoop-la', 'Advanced hoop-la']
-      }
+      const headers = new Headers()
+      headers.append('Authorization', 'Basic hoop-la')
+      headers.append('Authorization', 'Advanced hoop-la')
 
-      const res = fetchMock.mock('http://swagger.io', {headers})
+      xapp.get('http://swagger.io', {})
 
-      return fetch('http://swagger.io').then((_res) => {
+      fetch('http://swagger.io').then((_res) => {
         return serializeRes(_res, 'https://swagger.io')
       }).then((resSerialize) => {
         expect(resSerialize.headers).toEqual({authorization: ['Basic hoop-la', 'Advanced hoop-la']})
-      }).then(fetchMock.restore)
-    })
-
-    it('should set .text and .data to body Blob or Buffer for binary response', function () {
```

Comment:
sorry, that's an artifact of the way I tried to recover the files. I'll fix that in the morning.